src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -77,12 +57,12 @@ object JesBackend {
   }
 
   def workflowGcsPath(descriptor: WorkflowDescriptor): String = {
-    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, JesConf.getString("baseExecutionBucket"))
+    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, JesConf.executionBucket)
```

Comment:
So any of the JesBackend functionality which happens in the Object cannot be tested with a mock configuration and instead refer to the application.conf anyway? That... sounds dodgy to me. 

Maybe we can move it all into the class and remove the conf in the object completely?